modules/deploy/pages/deployment-option/self-hosted/manual/production/production-readiness.adoc

```diff
@@ -0,0 +1,534 @@
+
+Example:
+```bash
+rpk cluster config get kafka_enable_authorization
+```
+
+Output:
+```bash
+true
+```
+
+[[tls-enabled]]
+=== TLS Enabled 
+
+Ensure that all public interfaces have TLS enabled.
+
+Example:
+```bash
+grep tls /etc/redpanda/redpanda.yaml
+    kafka_api_tls:
+    admin_api_tls:
+    rpc_server_tls:
+        tls:
+        tls:
+    schema_registry_api_tls:
+```
+
+[[redpanda-tuners]]
+=== Redpanda Tuners
+
+Ensure that tuners have been run on all cluster hosts. Redpanda tuners ensure that the operating system is configured for optimal performance and can have a significant impact on the latency and throughput. In Kubernetes, you may need to run the tuners on the hosts themselves, rather than in containers.
+
+Example:
+```bash
+systemctl status redpanda-tuner
+```
+
+Output:
+```bash
+● redpanda-tuner.service - Redpanda Tuner
+     Loaded: loaded (/lib/systemd/system/redpanda-tuner.service; enabled; vendor preset: enabled)
+     Active: active (exited) since Mon 2024-03-25 12:03:51 UTC; 48min ago
+    Process: 3795 ExecStart=/usr/bin/rpk redpanda tune all $CPUSET (code=exited, status=0/SUCCESS)
+   Main PID: 3795 (code=exited, status=0/SUCCESS)
+
+Mar 25 12:03:51 ip-172-31-10-199 rpk[3795]: cpu                    true     true     true
+Mar 25 12:03:51 ip-172-31-10-199 rpk[3795]: disk_irq               true     true     true
+Mar 25 12:03:51 ip-172-31-10-199 rpk[3795]: disk_nomerges          true     true     true
+Mar 25 12:03:51 ip-172-31-10-199 rpk[3795]: disk_scheduler         true     true     true
+Mar 25 12:03:51 ip-172-31-10-199 rpk[3795]: disk_write_cache       false    true     false      Disk write cache tuner is only supported in GCP
+Mar 25 12:03:51 ip-172-31-10-199 rpk[3795]: fstrim                 false    false    true
+Mar 25 12:03:51 ip-172-31-10-199 rpk[3795]: net                    true     true     true
+Mar 25 12:03:51 ip-172-31-10-199 rpk[3795]: swappiness             true     true     true
+Mar 25 12:03:51 ip-172-31-10-199 rpk[3795]: transparent_hugepages  false    false    true
+Mar 25 12:03:51 ip-172-31-10-199 systemd[1]: Finished Redpanda Tuner.
+```
+
+Ensure that xref:reference:rpk/rpk-iotune.adoc[`rpk iotune run`] has been run on all hosts. Ensure that the mountpoint listed in this configuration file matches the mountpoint for Redpanda's data directory, usually configured at `/var/lib/redpanda`. See xref:deploy/deployment-option/self-hosted/manual/production/production-deployment.adoc#generate-optimal-io-configuration-settings[Generate optimal I/O configuration settings].
+
+Example:
+```bash
+cat /etc/redpanda/io-config.yaml
+disks:
+  - mountpoint: /mnt/vectorized
+    read_iops: 413115
+    read_bandwidth: 1882494592
+    write_iops: 182408
+    write_bandwidth: 788050688
+```
+
+[[disk-perf]]
+=== Run self test to check disk performance
+
+Run the self test to ensure that disk performance is within an acceptable range.
```

Comment:
In our gdoc we link to an external source. However we can't do that here. What are our acceptable ranges?

Reply:
We don't link anywhere.  That was referenced in a comment. we should not link to that, we should get Eng to assert what the acceptable range is.

Reply:
the link to running self test was dropped from this section, can we add back?

https://docs.redpanda.com/current/manage/cluster-maintenance/cluster-diagnostics/

Reply:
I don't see a range specified in the cluster diagnostics topic link here. So what is the acceptable range users should look for? We can't tell them to look for an "acceptable range" and then not tell them what that range is.